server/game/cards/08.3-Km/Jhiqui.js

```diff
@@ -0,0 +1,37 @@
+const DrawCard = require('../../drawcard.js');
+
+class Jhiqui extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                afterChallenge: ({challenge}) =>
+                    challenge.winner === this.controller && this.hasParticipatingLordOrLady()
+            },
+            cost: ability.costs.discardFromHand(),
+            target: {
+                activePromptTitle: 'Select a character',
+                cardCondition: card =>
+                    card.location === 'play area' && this.game.currentChallenge.isParticipating(card)
+            },
+            handler: context => {
+                context.target.modifyPower(1);
+                this.game.addMessage('{0} uses {1} and discards {2} to have {3} gain 1 power',
+                    this.controller, this, context.costs.discardFromHand, context.target);
+            }
+        });
+    }
+
+    hasParticipatingLordOrLady() {
+        let cards = this.controller.filterCardsInPlay(card => {
```

Comment:
This could be done more easily with:

```javascript
return this.controller.anyCardsInPlay(card => this.game.currentChallenge.isParticipating(card) && (card.hasTrait('Lord') || card.hasTrait('Lady'))
```

Also, technically you should be able to win a challenge, and then trigger Jhiqui to give an opponent's Lord or Lady 1 power, which should then be reflected in this check. I'm not sure if that's something we should worry about, but if you do, checking ```this.game.anyCardsInPlay()``` as above would take care of that.